Source/Fuse.Controls.Primitives/Behaviors/Shadow.uno

```diff
@@ -221,6 +234,82 @@ namespace Fuse.Controls
 			}
 		}
 
+		void AddNativeDecoration()
+        {
+			DecorateNativeShadow(Color, Size, Offset);
+        }
```

Comment:
Please fix indentation. Several methods in this file have mixed indentation.

Reply:
Oops!, fixed